Makefile.common

```diff
@@ -84,13 +84,15 @@ PROMU_URL     := https://github.com/prometheus/promu/releases/download/v$(PROMU_
 GOLANGCI_LINT :=
 GOLANGCI_LINT_OPTS ?=
 GOLANGCI_LINT_VERSION ?= v1.39.0
+ifeq ($(CIRCLE_JOB),)
```

Comment:
Why we need that? I don't see anything specifying this

Reply:
If we're in a CI pipeline that means that golangci is being run separately in an Action. We still want it to run with the Makefile locally.